.gitignore

```diff
@@ -2,33 +2,353 @@
 /build
 AssemblyInfo.cs
 
-# Standard VS.NET and ReSharper Foo
-.vs/*
-src/*/obj
-src/*/bin
-samples/*/obj
-samples/*/bin
-*.csproj.user
-*ReSharper.user
-_ReSharper*
-*.suo
 *.cache
 * Thumbs.db
 
 # Leftovers from merging
 *.orig
 *.bak
-*.sln.DotSettings.user
-*.DotSettings.user
-
-# Buildscripts artifacts
-buildscripts/**/obj/
-buildscripts/**/bin/
 
 # Tools artifacts
-tools/**/obj/
 tools/**/build/
```

Comment:
I haven't had a look but what is in `tools/**/build`? I'm going to guess the project was modified to write to build from the standard bin directory?